src/lang/en-US.ts

```diff
@@ -1,7 +1,7 @@
 // We DO NOT need this file sorted as it makes it easier for folks to update/see changes
 import { ResolvedIntlConfig } from 'react-intl/src/types';
 
-const enUSMessages: ResolvedIntlConfig['messages'] = {
+const CommonMessages: ResolvedIntlConfig['messages'] = {
```

Comment:
Was the deviation from camel case variable names intentional in this file? If so, how did you arrive at that conclusion?